awx/main/isolated/manager.py

```diff
@@ -210,10 +210,22 @@ def check(self, interval=None):
         if status == 'successful':
             status_path = self.path_to('artifacts', self.ident, 'status')
             rc_path = self.path_to('artifacts', self.ident, 'rc')
-            with open(status_path, 'r') as f:
-                status = f.readline()
-            with open(rc_path, 'r') as f:
-                rc = int(f.readline())
+            if os.path.exists(status_path):
+                with open(status_path, 'r') as f:
+                    status = f.readline()
+                with open(rc_path, 'r') as f:
+                    rc = int(f.readline())
+            else:
+                # if there's no status file, it means that runner _probably_
+                # exited with a traceback (which should be logged to
+                # daemon.log)  Record it so we can see how runner failed.
+                daemon_path = self.path_to('daemon.log')
+                if os.path.exists(daemon_path):
```

Comment:
If this fails for some reason, not much else we can do.

Reply:
Nope.